packages/perseus/src/widgets/radio/__stories__/choice-icon.stories.jsx

```diff
@@ -219,22 +219,28 @@ export const AllPositions = (args: StoryArgs): React.Node => {
         <>
             <p>Library</p>
             {allLetters.map((_, i: number) => (
-                <ChoiceIcon {...defaultProps} pos={i} />
+                <ChoiceIcon {...defaultProps} pos={i} key={"choice" + i} />
```

Comment:
I added these to keep the console clean of errors when looking at this component in Storybook.